opennms-base-assembly/src/main/filtered/etc/events/opennms.mib.events.xml

```diff
@@ -14,6 +14,7 @@
             <mevalue>1</mevalue>
          </maskelement>
       </mask>
+       <!-- TODO MVR can this go ?? -->
       <uei>uei.opennms.org/traps/eventTrap</uei>
```

Comment:
Does anyone know if I can remove this? The description states, it is related to scriptd, but I am not sure. @indigo423, @mhuot any ideas?

Reply:
I removed it for now. Feel free to leave any comment, so I can address it afterwards.